opensourceleg/actuators/dephy.py

```diff
@@ -0,0 +1,1739 @@
+        """
+        self.command_motor_voltage(value=int(value))
+
+    @deprecated_with_routing(alternative_func=set_motor_voltage)
+    def set_voltage(self, value: float) -> None:
+
+        self.command_motor_voltage(value=int(value))
+
+    def set_motor_position(self, value=float) -> None:
+        """
+        Sets the motor position in radians.
+        If in impedance mode, this sets the equilibrium angle in radians.
+
+        Args:
+            value (float): The position to set
+        """
+        self.command_motor_position(
+            value=int(
+                (value + self.motor_zero_position + self.motor_position_offset)
+                / self.MOTOR_CONSTANTS.RAD_PER_COUNT
+            ),
+        )
+
+    def set_position_gains(
+        self,
+        kp: float = DEFAULT_POSITION_GAINS.kp,
+        ki: float = DEFAULT_POSITION_GAINS.ki,
+        kd: float = DEFAULT_POSITION_GAINS.kd,
+        ff: float = DEFAULT_POSITION_GAINS.ff,
+    ) -> None:
+        """
+        Sets the position gains in arbitrary Dephy units.
+
+        Args:
+            kp (float): The proportional gain
+            ki (float): The integral gain
+            kd (float): The derivative gain
+            ff (float): The feedforward gain
+        """
+        self.set_gains(
+            kp=int(kp),
+            ki=int(ki),
+            kd=int(kd),
+            k=0,
+            b=0,
+            ff=int(ff),
+        )
+
+    def set_current_gains(
+        self,
+        kp: float = DEFAULT_CURRENT_GAINS.kp,
+        ki: float = DEFAULT_CURRENT_GAINS.ki,
+        kd: float = DEFAULT_CURRENT_GAINS.kd,
+        ff: float = DEFAULT_CURRENT_GAINS.ff,
+    ) -> None:
+        """
+        Sets the current gains in arbitrary Dephy units.
+
+        Args:
+            kp (float): The proportional gain
+            ki (float): The integral gain
+            kd (float): The derivative gain
+            ff (float): The feedforward gain
+        """
+        self.set_gains(
+            kp=int(kp),
+            ki=int(ki),
+            kd=int(kd),
+            k=0,
+            b=0,
+            ff=int(ff),
+        )
+
+    def set_motor_impedance(
+        self,
+        kp: float = DEFAULT_IMPEDANCE_GAINS.kp,
+        ki: float = DEFAULT_IMPEDANCE_GAINS.ki,
+        kd: float = DEFAULT_IMPEDANCE_GAINS.kd,
+        k: float = 0.08922,
+        b: float = 0.0038070,
+        ff: float = 128,
+    ) -> None:
+        """
+        Set the impedance gains of the motor in real units: Nm/rad and Nm/rad/s.
+
+        Args:
+            kp (float): Proportional gain. Defaults to 40.
+            ki (float): Integral gain. Defaults to 400.
+            kd (float): Derivative gain. Defaults to 0.
+            k (float): Spring constant. Defaults to 0.08922 Nm/rad.
+            b (float): Damping constant. Defaults to 0.0038070 Nm/rad/s.
+            ff (float): Feedforward gain. Defaults to 128.
+        """
+        self.set_impedance_gains(
```

Comment:
For example here, I think `set_impedance_gains` in this context should be `_set_impedance_gains`